[//]: # (This file is generated by eslint-docgen. Do not edit it directly.)

# file-names

showFilenames: true

## Rule details

❌ Examples of **incorrect** code:
```js
var vueX = 123;
var vueY = 456;
```

✔️ Examples of **correct** code:
```js
var x = 123;
var y = 456;
```

❌ Examples of **incorrect** code for a file named `test.js`:
```js
var tsX = 123;
var tsY = 456;
```

✔️ Examples of **correct** code for a file named `test.js`:
```js
var jsX = 123;
var jsY = 456;
```

## Resources

* [Rule source](/rules/file-names.js)
* [Test source](/tests/file-names.js)
