# my-rule

## Rule details

‚úîÔ∏è Examples of **correct** code:
```js
/*eslint my-plugin/my-rule: "error"*/
var x="123"
var y="45678"
```

‚ùå Examples of **incorrect** code with `[{"myOption":true}]` options:
```js
/*eslint my-plugin/my-rule: ["error",[{"myOption":true}]]*/
var x="1.23"
var y="4.5678"
```

üîß Examples of code **fixed** by using  `--fix` with `[{"myOption":true}]` options:
```js
/*eslint my-plugin/my-rule: ["error",[{"myOption":true}]]*/
var x="1.23" /* ‚Üí */ var x="123"
```

## Resources

* [Rule source](/rules/my-rule.js)
* [Test source](/tests/my-rule.js)
