[//]: # (This file is generated by eslint-docgen. Do not edit it directly.)

# my-rule

## Rule details

‚ùå Examples of **incorrect** code:
```js
var x = '1.23';
```

## Resources

* [Documentation source](/docs/my-rule.md)
