[//]: # (This file is generated by eslint-docgen. Do not edit it directly.)

# my-rule

## Rule details

❌ Examples of **incorrect** code:
```js
/* eslint @user/scoped/my-rule: "error" */
const x = '1.23';
```

✔️ Examples of **correct** code:
```js
/* eslint @user/scoped/my-rule: "error" */
const x = '4.56';
```

## Resources

* [Rule source](/rules/my-rule.js)
* [Test source](/tests/my-rule.js)
